mkdocs.yml

```diff
@@ -14,6 +14,7 @@ pages:
   - 'Course configuration': 'course.md'
   - 'Course instance configuration': 'courseInstance.md'
   - 'Assessment configuration': 'assessment.md'
+  - 'Assessment design': 'assessment-design.md'
```

Comment:
How about:

- `Assessment Configuration` -> `Overview of Assessment Options`
- `Assessment Design` -> `Designing an Assessment`